server/tests/output/formatter/jimi_test.py

```diff
@@ -517,7 +517,7 @@ def test_placeline_washington(self):
     def test_format_content(self):
         item = self.format_item(
             {
-                "body_html": "<p>Body HTML<br>test remove bold <b>  </b> and <b>bold1</b> and <i>idiom</i> and removed empty bold <b> </b></p>"
+                "body_html": "<p>Body HTML<br>test remove bold<b>  </b> and <b>bold1</b> and <i>idiom</i> and removed empty bold <b> </b></p>"
```

Comment:
you can keep this one, I suppose we should just do at the end some `.replace('  ', ' ')`

Reply:
okay, we can use the  `join()` function with the string `split()` function, which will remove all the unnecessary spaces.

Reply:
hm actually lets not do it, and don't bother with the extra space, they have some preformatted content with tabular like data using spaces so we would break that completely

Reply:
Okay, so currently I used the ` join()` function with the string `split()` function only in the` jimi_test` file so it will not break preformatted content.